services/QuillLMS/app/views/cms/users/_form.html.erb

```diff
@@ -39,14 +39,19 @@
         <%= f.select :admin_sub_role, [nil].concat(AdminInfo::SUB_ROLES) %>
       </div>
       <div class='cms-form-row'>
-        <%= f.label :admin_approval_status %>
+        <%= f.label 'Admin verification status' %>
```

Comment:
Any reason for the change from symbol to string here?  I doubt it matters, but I was curious.

Reply:
No, good question. I thought I needed it because the actual attribute is "approval_status", not "verification_status" here, but Peter Sharkey requested the copy change for consistency, but actually using the symbol works just fine.